basics/signals/src/panel.ts

```diff
@@ -9,20 +9,20 @@ const PANEL_CLASS = 'jp-RovaPanel';
 /**
  * A panel which contains a console and the ability to add other children.
  */
-export class TutorialPanel extends StackedPanel {
+export class ExamplePanel extends StackedPanel {
   constructor() {
     super();
     this.addClass(PANEL_CLASS);
-    this.id = 'TutorialPanel';
-    this.title.label = 'Tutorial View';
+    this.id = 'ExamplePanel';
+    this.title.label = 'Example View';
     this.title.closable = true;
 
-    this.tutorial = new TutorialView();
-    this.addWidget(this.tutorial);
-    this.tutorial.stateChanged.connect(() => {
+    this.widget = new ExampleView();
+    this.addWidget(this.widget);
+    this.widget.stateChanged.connect(() => {
       console.log('changed');
     });
   }
 
-  private tutorial: TutorialView;
+  private widget: ExampleView;
```

Comment:
If we follow the JupyterLab code style (#4), private variables should be prefixed by a `_`.

It's also fine if we do a pass on the style in a different PR.